src/item.cpp

```diff
@@ -1246,15 +1269,9 @@ std::string item::info(bool showtext, std::vector<iteminfo> *dump, bool debug) c
                 _("This piece of clothing is very fancy.")));
         }
         if (is_armor() && type->id == "rad_badge") {
-            size_t i;
-            for( i = 1; i < sizeof(rad_dosage_thresholds) / sizeof(rad_dosage_thresholds[0]); i++ ) {
-                if( irridation < rad_dosage_thresholds[i] ) {
-                    break;
-                }
-            }
             dump->push_back(iteminfo("DESCRIPTION",
                 string_format(_("The film strip on the badge is %s."),
-                              _(rad_threshold_colors[i - 1].c_str()))));
+                              _(rad_badge_color(irridation).c_str()))));
```

Comment:
Need to go the other way with this (leave initialized values translated and avoid translating here), as it is now the gettext system is not necessarily going to be able to extract the strings from the code.  In reality I'm pretty sure all the colors are covered elsewhere, but it's still bad form.